metadata_layer.py

```diff
@@ -550,7 +552,7 @@ def mirror_link(self, model_object, data_source, link, link_obj, policy):
         if edition and edition.title:
             title = edition.title
         else:
-            title = self.title or None
+            title = getattr(self, 'title', None) or None
```

Comment:
This is necessary now that we're calling `mirror_link` directly on MetaToModelUtility, which unlike `Metadata` doesn't have a `.title`. In theory this could have been a problem with `CirculationData` as well.